src/query/frontend/ast/cypher_main_visitor.cpp

```diff
@@ -1277,7 +1277,9 @@ antlrcpp::Any CypherMainVisitor::visitGrantPrivilege(MemgraphCypher::GrantPrivil
   auth->user_or_role_ = std::any_cast<std::string>(ctx->userOrRole->accept(this));
   if (ctx->privilegeList()) {
```

Comment:
duplication of code on 3 places (GRANT, REVOKE, DENY). I think we can use a lambda here.